ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/aaa-authorization.j2

```diff
@@ -1,15 +1,15 @@
 {# eos - aaa authorization #}
-{% if aaa_authorization is defined and aaa_authorization is not none %}
-{%     if aaa_authorization.exec.default is defined and aaa_authorization.exec.default is not none %}
+{% if aaa_authorization is arista.avd.defined  %}
```

Comment:
As arista.avd.defined does a deep test, this `if aaa_authorization is arista.avd.defined` should be removed

Reply:
! was missing. Added